specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -805,6 +907,56 @@
             "create",
             "read"
           ]
+        },
+        "additionalCapabilities": {
+          "type": "array",
+          "description": "List of additional capabilities to enable or disable for a Disk Pool",
+          "items": {
+            "$ref": "#/definitions/AdditionalCapability"
+          }
+        }
+      }
+    },
+    "DiskPoolUpdateProperties": {
+      "type": "object",
+      "description": "Properties section of the Disk Pool request payload.",
+      "properties": {
+        "provisioningState": {
+          "description": "State of the operation on the resource",
+          "type": "string",
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true
```

Comment:
As this is for input of diskpool update, the readonly properties should not be here.